packages/next/src/lib/typescript/writeConfigurationDefaults.test.ts

```diff
@@ -93,50 +93,31 @@ describe('writeConfigurationDefaults()', () => {
           // eslint-disable-next-line no-control-regex
           .replace(/\x1B\[\d+m/g, '') // remove color control characters
       ).toMatchInlineSnapshot(`
-      "
-        
-      We detected TypeScript in your project and reconfigured your tsconfig.json file for you. Strict-mode is set to false by default.
-        
-      The following suggested values were added to your tsconfig.json. These values can be changed to fit your project's needs:
-
-        
-      	- target was set to ES2017 (For top-level \`await\`. Note: Next.js only polyfills for the esmodules target.)
-        
-      	- lib was set to dom,dom.iterable,esnext
-        
-      	- allowJs was set to true
-        
-      	- skipLibCheck was set to true
-        
-      	- strict was set to false
-        
-      	- noEmit was set to true
-        
-      	- incremental was set to true
-        
-      	- include was set to ['next-env.d.ts', '.next/types/**/*.ts', '**/*.ts', '**/*.tsx']
-        
-      	- plugins was updated to add { name: 'next' }
-        
-      	- exclude was set to ['node_modules']
-
-        
-      The following mandatory changes were made to your tsconfig.json:
-
-        
-      	- module was set to esnext (for dynamic import() support)
-        
-      	- esModuleInterop was set to true (requirement for SWC / babel)
-        
-      	- moduleResolution was set to node (to match webpack resolution)
-        
-      	- resolveJsonModule was set to true (to match webpack resolution)
-        
-      	- isolatedModules was set to true (requirement for SWC / Babel)
-        
-      	- jsx was set to preserve (next.js implements its own optimized jsx transform)
-      "
-    `)
+        "
+           We detected TypeScript in your project and reconfigured your tsconfig.json file for you. Strict-mode is set to false by default.
```

Comment:
Verified the change is still correct, snapshot only changes the indentation here but not sure why

![image](https://github.com/user-attachments/assets/bf4f80db-0aa5-45f8-8a8f-411aec2bb108)

Reply:
I haven't figured out when it does that either. I think at some point they changed the indentation default and then snapshots just weren't touched. You should be able to restore the indentation to before and the snapshots still match iirc.

Reply:
Only the tsconfig json snapshot can be recovered, this one will still fail if recover the indentation